deployment.md

```diff
@@ -20,61 +17,19 @@ Note:
 		+ "dummy" computers doing the hard work (compiling, testing)
 
-
 ## Get some machines
 + You need some machines to do testing.
 + Usually you want to have different systems to do tests.
 + Theoretically you can use bare metal machines, virtual machines, containers etc.
-
-### How to get machines on OpenStack
-1. (Get OpenStack acc and login)
-1. *Import your key* (OS: Access & Security / Key Pairs tab) before you create any machine!
-    + Usually you don't know any password on the virtual machine
-    + This option injects your keys into machine during its creation process.
-    + Might be good idea to import key from Master node
-1. Create master and slaves machines (example)
-    + OS: Instances =>
-        + Launch Instance
-            + Flavor: m1.large
-        + Boot from image
-            + *Fedora cloud 22*
-        + Launch
-1. Now you have machine somewhere in the cloud and you have to get access to the machine:
-    + Get *public IP* (at least for master)
-        + OS: Access & Security / Floating IPs > Allocate IP to Project
-        + OS: Select IP -> Associate -> select instance
-    + *Manage access to machines* (= open ports)
-        + OS: Access & Security / Security Groups
-            + You probably need to create Rules (OS: Groups/Rules) which enable you access to *ssh*, *http*s.
-        + You can restrict SSH only for Red Hat Subnet etc.
-        + **Jenkins** master needs (ssh/tcp) :56917 port allowed
-            + You can connect to this port using ssh.
-            + Jenkins provides you tunnel to slaves(last build, etc) via this port.
++ We currently run Jenkins on AWS
 
 ### How to connect to the machine
-+ Note: depending on the image used for installation, node might be created with user other than 'cloud-user'. Check console output, and search for part where public key is imported to get a username.
-#### Connect to your Master
-Master has public IP address, so simply using ```ssh cloud-user@<master ip>``` should be enough. 'cloud-user' has your public key imported, and has sudo access.
-
-#### Connect to a slave machine
-Slave machines usually do not hold public IP, Master node has access to them, though. That means there are at least three ways how to access the machine.
-1. Assign *public* (floating) *IP* to our slave using OpenStack
-1. Use *port forwarding* from master to slave (use your public key)
-    + use ```ssh -L 2222:<slave-ip>:22 cloud-user@<master-ip>``` forward ssh port to slave
-    + connect to slave via redirected port ```ssh cloud-user@localhost -p 2222```
-1. Connect directly *from master to slave*
-    + for this, we need to import master public key into slave authorized_keys first
-
+We use Ansible for orchestration of the entire Jenkins infrastructure. Please see the
+inventory.ini file for a list of machines and their IPs. These machines currently run
+in AWS. If you want your public key added to those machines please contact the admin
+that maintains the AWS account.
```

Comment:
It's not a huge risk to publish a list of ssh public keys but it makes it easier to lookup leaked keys. I have a private playbook that authorizes all the keys in it on all the machines so I just use that. I can share that privately if you want.